pkg/networkutils/network.go

```diff
@@ -340,9 +416,13 @@ func GetConfigForDebug() map[string]interface{} {
 	}
 }
 
-// useExternalSNAT returns whether SNAT of secondary ENI IPs should be handled with an external
+// UseExternalSNAT returns whether SNAT of secondary ENI IPs should be handled with an external
 // NAT gateway rather than on node.  Failure to parse the setting will result in a log and the
 // setting will be disabled.
+func (n *linuxNetwork) UseExternalSNAT() bool {
+	return getBoolEnvVar(envExternalSNAT, false)
+}
```

Comment:
Nit. Could it just be `return useExternalSNAT()` instead, since they do the same thing? Or should the env variable be saved as a property on the `linuxNetwork`?